scripts/update-component-sha.sh

```diff
@@ -1,10 +1,66 @@
 #!/bin/sh
 set -e
-if [ $# -ne 2 ] ; then
-    echo "Need <OLD> and <NEW> as arguments" >&2
+
+##
+#
+# script to replace hashes in config files
+# see usage() for usage and functionality
+#
+
+function usage() {
+    echo "$0 --<mode> <how-to-find> <new-hash>" >&2
+    echo >&2
+    echo "Avaioable modes: --hash and --image" >&2
+    echo >&2
+    echo "Replace by hash:" >&2
+    echo "	$0 --hash <OLD> <NEW>" >&2
+    echo "	Example: $0 8675309abcdefg abcdef567899" >&2
+    echo "	   Will replace all instances of 8675309abcdefg with abcdef567899" >&2
+    echo >&2
+    echo "Replace by image: $0 --image <IMAGE> <NEW>" >&2
+    echo "	$0 --image <IMAGE> <NEW>" >&2
+    echo "	Example: $0 linuxkit/foo abcdef567899" >&2
+    echo "	   Will tag all instances of linuxkit/foo with abcdef567899" >&2
+    echo >&2
+    echo "By default, for convenience, if no mode is given (--image or --hash), the furst method (--hash) is assumed. Thus the following are equivalent:" >&2
```

Comment:
I thought of that but was too lazy, since it started as one `echo`, then two, then...

I guess I should fix it.